cmd/serf/command/agent/command.go

```diff
@@ -156,6 +171,13 @@ func (c *Command) readConfig() *Config {
 		config.RetryInterval = minRetryInterval
 	}
 
+	// Check for sane broadcast timeout
+	if config.BroadcastTimeout < minBroadcastTimeout {
+		c.Ui.Output(fmt.Sprintf("Warning: 'BroadcastTimeout' is too low. Setting to %v",
+			config.BroadcastTimeout))
+		config.BroadcastTimeout = minBroadcastTimeout
```

Comment:
Oops, I blindly followed "RetryInterval" above. OK if I fix that for RetryInterval as well?